templates/terraform/examples/cloudbuild_trigger_github.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
+  trigger_template {
+    branch_name = "master"
+    repo_name   = "my-repo"
+  }
+
+  substitutions = {
+    _FOO = "bar"
+    _BAZ = "qux"
+  }
+
+  github {
+    installation_id = "abc123"
```

Comment:
Will this be auto populated or must we specify an `installation_id` ?

Reply:
Yes, that will be a required field.

Reply:
This field is actually ignored, but was missed for cleanup prior to the beta release. Feel free to ignore this field. I'll make sure this gets marked as deprecated and eventually cleaned up.

Reply:
Ah, that's wonderful.  Thanks!  I'll remove it.